### PR TITLE
Add question explaining onion.torproject.org

### PR DESCRIPTION
**Pull request by RotationMatrix**
Fix ticket #24.

### CODE REVIEW

File: content/onionservices/onionservices-4/contents.lr [hunk 1]
```diff
@@ -0,0 +1,8 @@
+_model: question
+---
+title: Does the Tor Project run any onion services?
```
Comment (gusgustavo) [inline content/onionservices/onionservices-4/contents.lr]: Onion Services

File: content/onionservices/onionservices-4/contents.lr [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+---
+seo_slug: tor-project-onion-services
+---
+description:
+Yes! A list of our onion services is available at <a href="https://onion.torproject.org/">onion.torproject.org</a>.
```
Comment (gusgustavo) [inline content/onionservices/onionservices-4/contents.lr]: Use markdown for links, e.g., [onion.torproject.org](https://onion.torproject.org)

**Review by gusgustavo — CHANGES_REQUESTED** [review]

File: content/onionservices/onionservices-4/contents.lr (hunk 1 above)
Comment (RotationMatrix) [reply content/onionservices/onionservices-4/contents.lr]: Capitalize all uses or only this one?

Comment (gusgustavo) [reply content/onionservices/onionservices-4/contents.lr]: All uses

Comment (RotationMatrix) [reply content/onionservices/onionservices-4/contents.lr]: You got it!

**Pull request merged**